pages/linux/screenkey.md

```diff
@@ -0,0 +1,32 @@
+# screenkey
+
+> A screencast tool to display keys pressed.
+> More information: <https://www.thregr.org/~wavexx/software/screenkey/>.
+
+- Display keys which are currently being pressed on the screen:
+
+`screenkey`
+
+- Display keys and mouse buttons which are currently being pressed on the screen:
+
+`screenkey --mouse`
+
+- Launch the settings menu of screenkey:
+
+`screenkey --show-settings`
+
+- Launch screenkey at a specific position:
+
+`screenkey --position {{top|center|bottom|fixed}}`
+
+- Change the format of the key modifiers displayed on screen:
+
+`screenkey --mods-mode {{normal|emacs|mac|win|tux}}`
+
+- Change the appearance of screenkey:
+
+`screenkey --bg-color "{{#353535}}" --font {{Hack}} --font-color {{yellow}} --opacity {{0.8}}`
```

Comment:
```suggestion
`screenkey --bg-color "{{#3b3b3b}}" --font {{Hack}} --font-color {{yellow}} --opacity {{0.8}}`
```
A hex code without a single letter kind of doesn't look right 😄

Reply:
I think it should be #FFFFFF

Reply:
How would that be a benefit? I think `3b3b3b` is astethically more pleasing than `FFFFFF`.

Reply:
How about `#A1B2C3`?

Reply:
This would prabably be the most generic variant. However I prefer `#a1b2c3`.

Reply:
Yeah, I also prefer `#a1b2c3`.
```suggestion
`screenkey --bg-color "{{#a1b2c3}}" --font {{Hack}} --font-color {{yellow}} --opacity {{0.8}}`
```

Reply:
That auto hex to color is pretty nice!